pkg/apis/monitoring/v1/types.go

```diff
@@ -657,6 +657,8 @@ type AlertmanagerEndpoints struct {
 	// Version of the Alertmanager API that Prometheus uses to send alerts. It
 	// can be "v1" or "v2".
 	APIVersion string `json:"apiVersion,omitempty"`
+	// Per-target Alertmanager timeout when pushing alerts.
```

Comment:
Nit: Can you instead do:
```suggestion
	// Timeout is a per-target Alertmanager timeout when pushing alerts.
```

Reply:
Is this a v1 Alertmanager field only?

Reply:
> Is this a v1 Alertmanager field only?

it also supports in v2